### PR TITLE
Minor Corrections to Documentation Content

### PR DESCRIPTION
**Pull request by Denton-L**
Small corrections to grammar, capitalisation, formatting, syntax errors and incorrect information.

### CODE REVIEW

File: docs/contracts.rst [hunk 1]
```diff
@@ -285,10 +293,12 @@ inheritable properties of contracts and may be overridden by derived contracts.
         mapping (address => bool) registeredAddresses;
         uint price;
         function Register(uint initialPrice) { price = initialPrice; }
-        function register() costs(price) {
+
+        function Register() costs(price) {
```
Comment (chriseth) [inline docs/contracts.rst]: I think this is lowercase on purpose, it is not the constructor.

**Comment by chriseth** [comment]
Thanks a lot, fine apart from the one tiny change.

**Comment by Denton-L** [comment]
Alright, fixed!

**Comment by chriseth** [comment]
Sorry, you need to rebase now

**Comment by Denton-L** [comment]
Merge conflicts fixed!

**Comment by chriseth** [comment]
Can you rebase on top of latest develop? Then we get nice doc build results from travis.

**Comment by Denton-L** [comment]
Sorry, I hadn't noticed that develop was recently updated.

File: docs/units-and-global-variables.rst [hunk 2]
```diff
@@ -94,8 +94,8 @@ Mathematical and Cryptographic Functions
     compute the SHA-256 hash of the (tightly packed) arguments
 `ripemd160(...) returns (bytes20)`:
     compute RIPEMD-160 hash of the (tightly packed) arguments
-`ecrecover(bytes32, uint8, bytes32, bytes32) returns (address)`:
-    recover public key from elliptic curve signature - arguments are (data, v, r, s)
+`ecrecover(bytes32 data, uint8 v, bytes32 r, bytes32 s) returns (address)`:
+    recover the address assocaited with the public key from elliptic curve signature
```
Comment (chriseth) [inline docs/units-and-global-variables.rst]: Typo: `assocaited`

**Comment by chriseth** [comment]
Found another one - sorry ;-)

**Comment by Denton-L** [comment]
No worries, my bad for being careless with my commits!

**Pull request merged**